FtcRobotController/src/main/java/org/firstinspires/ftc/robotcontroller/external/samples/ConceptTensorFlowObjectDetectionWebcam.java

```diff
@@ -49,7 +50,7 @@
  * IMPORTANT: In order to use this OpMode, you need to obtain your own Vuforia license key as
  * is explained below.
  */
-@TeleOp(name = "Concept: TensorFlow Object Detection Webcam", group = "Concept")
+@Autonomous(name = "Concept: TensorFlow Object Detection Webcam", group = "Concept")
```

Comment:
Please don't make changes outside your 417 folder

Reply:
Alright, we changed this. 😁

Reply:
It's not ok that you changed this outside your 417 folder. Change it back.

Reply:
We forgot to push our changes, that's why you didn't see the change before. Sorry! 😭>:(

Reply:
We have fixed this!